src/daft-table/Cargo.toml

```diff
@@ -4,14 +4,21 @@ comfy-table = {workspace = true}
 common-error = {path = "../common/error", default-features = false}
 daft-core = {path = "../daft-core", default-features = false}
 daft-dsl = {path = "../daft-dsl", default-features = false}
+daft-image = {path = "../daft-image", default-features = false}
```

Comment:
we can likely drop this, if we perform the `trait + register` pattern discussed above